docs/Stats/breakdown.md

```diff
@@ -30,8 +30,10 @@ and inside an institution scope they have the format:
 /{institution}/breakdown/{granularity}{counter}/{item}/{item_id}
 ```
 
-where **granularity** is one of `year`, `month`, `day` or `total`
-and **counter** is one of `views`, `downloads` or `shares`.
+where **granularity** is one of `year`, `month`, `day` or `total`,
```

Comment:
Should we also specify what the institution is, for example is it the **ID** or the **StringID**?